webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1233,6 +1282,118 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
             zipOut.putNextEntry(null, parameters);
             zipOut.write(join(stripNewlines(scrambleRequest.getAllScrambles()), "\r\n").getBytes());
             zipOut.closeEntry();
+                
+            parameters.setFileNameInZip("Internationalization/");
```

Comment:
What's the point of this line?

Reply:
I set it to create the folder when I was learning about how tnoodle handled pdfs. I'll supress the line.